service/DevDocs/SendAsyncTimeout.md

```diff
@@ -0,0 +1,36 @@
+# SendAsyncTimeout Requirements
+
+## Overview
+
+DeviceClient class is used by the device client callback delegate to prepare the callback results.
```

Comment:
Is this correct? The code below is referring to the AmqpServiceClient.

Reply:
Copy and paste shrapnel. Will be fixed with next iteration